PyTDF/Node.py

```diff
@@ -31,7 +31,7 @@ def _call_handler(self, *args, **kwargs):
         newNode = Node(operation=op, _get_head=self._get_head)
         self.next_nodes.append(newNode)
 
-        if op.op_type == "a":
+        if op.op_type == op.TYPES.ACTION:
```

Comment:
Same comments as above: access with the class and lowercase for the enum type name